apps/src/lab2/lab2Redux.ts

```diff
@@ -135,14 +136,28 @@ export const setUpWithLevel = createAsyncThunk(
           : await ProjectManagerFactory.getProjectManagerForLevel(
               ProjectManagerStorageType.REMOTE,
               payload.levelId,
+              payload.userId,
               payload.scriptId
             );
+
       // Only set the project manager and initiate load
       // if this request hasn't been cancelled.
       if (thunkAPI.signal.aborted) {
         return;
       }
 
+      // We might be a teacher attempting to view a student level that hasn't been
+      // started, and there is no project manager available.
```

Comment:
Just pointing out that this will be useful for other cases where we don't want to load project as well, such as editing start sources (which @mikeharv will be working on)